### PR TITLE
Backend feature/home-category

### LINKED ISSUE #158: Implementation of home page end category endpoints
Endpoints '/' and '/category' will be created. They will be used to get product lists from the given category.

### PR DESCRIPTION
**Pull request by lighterm**
# Description

Implemented the initial '/' and '/category' endpoints for the backend.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- Not tested with unit tests since this endpoint is subject to further changes.
- However, the functionality is tested manually with Postman.

**Test Configuration**:
* OS: MacOS
* Python: 3.9.0
* Dockerized: No

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: tursu/tursu/urls.py [hunk 1]
```diff
@@ -19,4 +19,7 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('user/', include('registered_user.urls')),
+    path('search/', include('search.urls')),
+    path('category', include('category.urls')),
```
Comment (sezgineray) [inline tursu/tursu/urls.py]: Slash is forgotten at the end of category?

**Review by sezgineray — CHANGES_REQUESTED** [review]
Is there a need for a separate _category_ app? I believe its models and views should be under product.

File: tursu/tursu/urls.py (hunk 1 above)
Comment (lighterm) [reply tursu/tursu/urls.py]: Yes, I just noticed I was sending the HTTP requests like "http://127.0.0.1:8000/category?name=Electronics" which is wrong. I just fixed it.

**Comment by sezgineray** [comment]
Code appears to be ready for merge after resolving conflicts.

**Pull request merged**